packages/components/icon/src/Icon.tsx

```diff
@@ -74,6 +74,7 @@ export type IconInternalProps = CommonProps & {
   size?: IconSize;
   /**
    * Whether or not to trim the icon width, i.e. set `width` to `auto`
+   * @deprecated IconsV5 will no longer support the trimmed option.
```

Comment:
```suggestion
   * @deprecated Trimmed icons will be removed in a future major release
```

I think we should be less specific here, haha 😅